frontend-new/src/chat/chatProgressbar/getUserFriendlyConversationPhaseName.ts

```diff
@@ -1,26 +1,29 @@
 import { ConversationPhase, CurrentPhase } from "./types";
-
-export const USER_FRIENDLY_PHASE_NAMES: Record<ConversationPhase, string> = {
-  [ConversationPhase.INITIALIZING]: "Initializing...",
-  [ConversationPhase.INTRO]: "Introduction",
-  [ConversationPhase.COLLECT_EXPERIENCES]: "Collecting",
-  [ConversationPhase.DIVE_IN]: "Exploring",
-  [ConversationPhase.ENDED]: "Conversation finished",
-  [ConversationPhase.UNKNOWN]: "Unknown phase",
-};
+import i18n from "src/i18n/i18n";
+// Compute translations at call time so they react to language changes and avoid module-scope resolution.
+export const getUserFriendlyPhaseNames = (): Record<ConversationPhase, string> => ({
```

Comment:
I would suggest, we go with `get() {...}`, so that if the user changes the language, it will affect.

This will be generated at load time,